bpfctl/Cargo.toml

```diff
@@ -14,7 +14,7 @@ path = "src/main.rs"
 anyhow = { workspace = true, features = ["std"] }
 base64 = { workspace = true }
 bpfd-api = { workspace = true }
-clap = { workspace = true, features = ["derive", "std"] }
+clap = { workspace = true, features = ["default", "derive"] }
```

Comment:
Can we not use `default`? What's the minimal feature set we need from `clap` for help to work?

Reply:
I think we all thought that the other features in default looked like they'd be nice to have:
std
color
help
usage
error-context
suggestions

Reply:
If we want them, we should add them explicitly - it could be that clap decide to change what's in "default" when they next do a release and remove something we rely on. This happened in the `nix` crate recently, but thankfully dependabot caught it.

Reply:
I think we need all 7 features. I can use all 7 if you prefer.